test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -1,14 +1,14 @@
-test_duration: 180
-stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5"
+test_duration: 30
```

Comment:
what are these changes?

Reply:
@roydahan
I am changing and check it according to conversation with Bentsi. Please, don't review now